WTWSMS/localisation/WtWSMS_misc.csv

```diff
@@ -224,3 +223,5 @@ rejected_senate;Rejected by Senate;R
 rejected_senate_desc;This character has received the rejected of the Senate.;Ce personnage a �t� r�fut� par le S�nat. ;;;Esta persona ha rechazado al Senadp;;;;;;x
 #### Trade routes;;;;;;;;;;;;;;x
 amber_route;Amber Road;Route de l'ambre;;;;;;;;;;;;x
+#### Misc misc;;;;;;;;;;;;;;x
+String_Requiescat_in_pace;Requiescat in pace;;;;;;;;;;;;;x
```

Comment:
Wrong file, this isn't added by WtWSMS.

Reply:
It's added by me. The string is missing even in vanilla.

Reply:
Then I would add it in WtWSMS_override.csv.

Reply:
ok, moved

Reply:
ok, moved